experimental/dds/tree/package.json

```diff
@@ -68,7 +68,7 @@
 		"test:coverage": "c8 npm test",
 		"test:mocha": "mocha \"dist/**/*.tests.js\" --exit -r node_modules/@fluid-internal/mocha-test-setup",
 		"test:mocha:verbose": "cross-env FLUID_TEST_VERBOSE=1 npm run test:mocha",
-		"test:stress": "cross-env FUZZ_TEST_COUNT=10 FUZZ_STRESS_RUN=true mocha \"dist/**/*.fuzz.tests.js\" --exit -r node_modules/@fluid-internal/mocha-test-setup",
+		"test:stress": "cross-env FUZZ_TEST_COUNT=10 FUZZ_STRESS_RUN=true mocha \"dist/**/*.fuzz.tests.js\" --exit -r @fluid-internal/mocha-test-setup",
```

Comment:
Merging this seems good, but I'll note that the .mocharc.cjs file in this package (and most packages?) do an explicit require of the config from mocha-test-setup, and that one [also adds the whole mocha-test-setup package to the list of required modules](https://github.com/microsoft/FluidFramework/blob/15531e2130f4b3c27198c2e9e887dc4829a30be5/packages/test/mocha-test-setup/mocharc-common.cjs#L14-L18). So _maybe_ all this `-r @fluid-internal/mocha-test-setup` might be unnecessary across the repo. Something we can try separately.